pkg/datarepair/datarepair_test.go

```diff
@@ -68,17 +70,25 @@ func TestDataRepair(t *testing.T) {
 		remotePieces := pointer.GetRemote().GetRemotePieces()
 		minReq := redundancy.GetMinReq()
 		numPieces := len(remotePieces)
-		toKill := numPieces - (int(minReq) + 1)
+		// disqualify one storage node
+		toDisqualify := 1
+		toKill := numPieces - (int(minReq+1) + toDisqualify)
 		// we should have enough storage nodes to repair on
-		assert.True(t, (numStorageNodes-toKill) >= numPieces)
+		assert.True(t, (numStorageNodes-toKill-toDisqualify) >= numPieces)
```

Comment:
I think this one should be `require` instead of `assert`. If this check is not satisfied then we have a bad test setup and we don't really want the test to continue.